features/instrumentationLogReader/src/main/java/org/opennms/util/ilr/ServiceCollector.java

```diff
@@ -56,7 +56,7 @@ public String getServiceID() {
         return m_serviceID;
     }
 
-    private static final String SERVICE_ID_REGEX = "([^/]+)/(\\d+)/(\\d+.\\d+.\\d+.\\d+)/([\\w-_]+)"; // Should be consistent with BaseLogMessage.s_regexp
+    private static final String SERVICE_ID_REGEX = "([^/]+)/(\\d+)/(\\d+.\\d+.\\d+.\\d+|[0-9a-fA-F:]+)/([\\w-_]+)"; // Should be consistent with BaseLogMessage.s_regexp
```

Comment:
https://stackoverflow.com/a/17871737/20273671

😅

Reply:
Yeah, we aren't strictly checking for only valid IPs, but since this is only for parsing a specific log file, there's the **assumption** that the content contains only valid IPs.